src/components/PlioQuestion.vue

```diff
@@ -69,13 +64,24 @@
           <i class="fas fa-times-circle" ref="wrong-icon"
             v-if="isAnswerSubmitted && !isAnswerCorrect"></i>
 
+          <!-- submit button -->
           <button
+            v-if="!isAnswerSubmitted"
```

Comment:
submit button is shown only when the answer has not been submitted yet.
If answer is submitted, replace it with done button